std/string.d

```diff
@@ -5316,9 +5320,13 @@ if (isSomeString!S)
 
 
 /***********************************************
+ * $(RED This function is deprecated and will be removed May 2018.)
+ * Please use the functions in $(MREF std, regex) and $(MREF std, algorithm)
+ * instead.
+ *
  * See if character c is in the intersection of the patterns.
  */
-
+deprecated("The std.string pattern functions are obsolete and will be removed May 2018. Use std.regex instead")
```

Comment:
That would be the wrong comment. Ideally, a comment like @@@DEPRECATED_2018-05@@@ is there so that it indicates when the function will be removed from the documentation, and I can grep for it. But lines with "Explicitly undocumented" are only for functions which are actually undocumented.

The normal procedure is to deprecate the function and mark it as deprecated in the documentation. Then after it's been deprecated for a year, the documentation is removed, and a comment like like the one you listed is put there instead. Then, after another year, the function is fully removed.